Site picker crashes after login if device is rotated

When I reach the site picker after logging in, I can consistently get the app to crash by rotating it a couple of times.

```
Caused by: java.lang.IllegalStateException: Trying to log an event step: site_list with a missing flow
        at com.woocommerce.android.ui.login.UnifiedLoginTracker.handleMissingFlowOrStep(UnifiedLoginTracker.kt:92)
        at com.woocommerce.android.ui.login.UnifiedLoginTracker.track(UnifiedLoginTracker.kt:42)
        at com.woocommerce.android.ui.login.UnifiedLoginTracker.track$default(UnifiedLoginTracker.kt:30)
        at com.woocommerce.android.ui.sitepicker.SitePickerActivity.showStoreList(SitePickerActivity.kt:322)
        at com.woocommerce.android.ui.sitepicker.SitePickerActivity.onCreate(SitePickerActivity.kt:165)
```

--- a/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerActivity.kt
+++ b/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/sitepicker/SitePickerActivity.kt
@@ -217,7 +217,7 @@ class SitePickerActivity :
             unifiedLoginTracker.getFlow()?.value?.let {
                 outState.putString(KEY_UNIFIED_TRACKER_FLOW, it)
             }
-            outState.putString(KEY_UNIFIED_TRACKER_FLOW, unifiedLoginTracker.getSource().value)
+            outState.putString(KEY_UNIFIED_TRACKER_SOURCE, unifiedLoginTracker.getSource().value)
             outState.putString(KEY_UNIFIED_TRACKER_STEP, unifiedLoginTracker.currentStep?.value)
         }
 

